src/_guides/libraries/library-tour.md

```diff
@@ -984,6 +974,49 @@ Future objects appear throughout the Dart libraries, often as the object
 returned by an asynchronous method. When a future *completes*, its value
 is ready to use.
 
+
+#### Basic usage
```

Comment:
This subsection isn't new, I just moved it up so that basic usage (explaining `Future.then()` etc) comes just before "Using await".

Reply:
FWIW, I'm not sure this is a good move. Most developers, most of the time, will want to use `async/await`. Maybe we want to call this section "low-level usage" and move it back?

Reply:
That's what I figured, but the presentation seemed off on first reading. I.e., the "Using await" section, when first, uses an example of "Basic usage" to motivate the use of await.

Anyhow, I'll revert to the original order.